website/pages/docs/platform/k8s/helm/examples/standalone-tls.mdx

```diff
@@ -120,9 +75,7 @@ TMPDIR=/tmp
 
     1. Create a file `${TMPDIR}/csr.yaml` with the following contents:
 
-        ```
-        $ export CSR_NAME=vault-csr
-        $ cat <<EOF >${TMPDIR}/csr.yaml
+        ```yaml
```

Comment:
```suggestion
        ```
        $ export CSR_NAME=vault-csr
        $ cat <<EOF >${TMPDIR}/csr.yaml
```

Reply:
Re-copied from existing examples to resolve rebasing issues.